PWGLF/Tasks/Resonances/xi1530Analysisqa.cxx

```diff
@@ -48,35 +48,44 @@ enum {
   kAllType
 };
 
+enum {
+  PionPID = 211,
+  XiPID = 3312,
+  XiStarPID = 3324
+};
```

Comment:
This is quite useless. The Pi and Xi codes already have their named constants. Use them. Xi* can be added in `Common/Constants/include/CommonConstants/PhysicsConstants.h` if needed.

Reply:
Thank you for your comments vit! 

I’ve made the updates after pulling the PR branch, but I accidentally ran a regular build instead of a ninja build, so it seems difficult to apply the updates right now ☹.

I will incorporate the changes in the next PR!

Also, if possible, could you add the Xi* code to PhysicsConstants.h?

Since I haven’t worked directly with O2 after forking it, it might be challenging for me to apply the changes immediately myself.

Thank you!